lib/rester/client/adapters/adapter.rb

```diff
@@ -37,10 +37,10 @@ def connected?
       end
 
       ##
-      # Sends an HTTP request to the service.
+      # Sends a request (using one of the subclass adapters) to the service.
       #
       # `params` should be a hash if specified.
-      def request(verb, path, params = nil)
+      def request(verb, path, params=nil)
```

Comment:
The recommended style is actually to have spaces around the `=`:
https://github.com/bbatsov/ruby-style-guide#spaces-around-equals

I didn't used to do it that way, but I started to put spaces around.